CHANGELOG.md

```diff
@@ -20,6 +20,7 @@ All notable changes to the Pony compiler and standard library will be documented
   (which may result in a copy) while `cpointer()` (also available on
   `Array` objects) returns a pointer to the underlying array as-is
   (issue #1309).
+- Ensuring TCP connection is established before writing data to it (issue #1310)
```

Comment:
Small nit, this should be "Ensure TCPConnection is established before writing data to it (issue #1310)"